site/content/integrate/marketplace-submissions/_index.md

```diff
@@ -0,0 +1,60 @@
+
+## Type of integrations to submit to the web marketplace
+
+- Apps
+- Plugins
+- Webhook configuration
+- Playbook templates
+- Utility tools, such as importers, command line interfaces, and scripts
+
+## How to submit your contribution for consideration for the web marketplace
+
+Please fill out the information in [this form](https://forms.gle/PE8kmfSuneP9GWnq8) to be reviewed by a member of the Mattermost team. 
+
+Every contribution will go through the following checklist before being added to the web marketplace at mattermost.com/marketplace:  
+
+1. There is a link to the contribution so we can test it out.
+2. There is documentation to support people installing, configuring, and using the contribution. We also recommend including screenshots to give users a better understanding of the workflow and user experience. 
+3. There is a public issue or bug tracker for users to report bugs or issues they encounter. 
+
+We will also reach out to you to learn more about the integration, your experience developing with Mattermost, and to coordinate postings for social media. 
+
+## How to submit your contribution for consideration for the in-product marketplace
+
+There are additional functional, technical, and security requirements for apps and plugins to be considered for the in-product marketplace. You can check off these items when you submit your contributions by opening an [issue in our in-product marketplace repository](https://github.com/mattermost/mattermost-marketplace/issues/new?assignees=hanzei&labels=Plugin%2FNew&template=add_plugin.md&title=Add+%24REPOSITORY_NAME+to+Marketplace). 
+
+### Technical requirements
+
+1. The plugin or app works for 60,000 concurrent connections and within a High Availability deployment. **Note:** There are currently no publicly-available tools available to verify these properties. As such, they are checked during code review by a developer.
+2. The plugin or app logs important events on appropriate log levels to allow System Admins to troubleshoot issues.
+
+### Security requirements
+
+1. Security reviews do not reveal any exploitable vulnerabilities.
+2. An email address or a username on the [Community Server](https://community.mattermost.com) is provided to be used to report vulnerabilities in the future.
+
+### Functional requirements
+
+1. The app or plugin must set a `min_server_version` in the manifest.
+2. The app or plugin must work on all Mattermost versions greater than or equal to the `min_server_version`.
+
+Please note that Mattermost reserves the right to reject any plugin submission from the Marketplace.
+
```

Comment:
This is missing [Product and Documentation requirements](https://github.com/mattermost/mattermost-marketplace/blob/master/.github/ISSUE_TEMPLATE/add_plugin.md). To reduce duplicity, I think these sections above could be removed and we could link to the issue template *.md instead.